### ISSUE TITLE
`:tx-listen` for Om Next

### ISSUE THREAD

**Issue by swannodette** [omcljs/om#384]
Currently we just use Google Closure logging. 3rd party tooling will probably want to tap directly into the transaction listening.

**Comment by bensu** [comment]
FWIW I've successfully used `ITxIntercept` on the reconciler to achieve this:

```clj
(specify! reconciler
  om/ITxIntercept
  (tx-intercept [this tx]
    (log-action! tx)
     tx))
```

**Comment by anmonteiro** [comment]
fixed ee7f9f60da20a1d5e4ee1b4dbcf8f7a709d833c5
